src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -101,7 +103,7 @@
   <ItemGroup>
     <EmbeddedResource Update="EditorFeaturesResources.resx" GenerateSource="true" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
```

Comment:
📝 This condition is required because we don't have a compatible set of dependencies yet

Reply:
Worked around this limitation by forcing the netcoreapp3.1 build to use the net472 build artifacts.